### PR TITLE
DEV-11823 : Updated Android SDK from 5.1.1 to 6.4.0

### PR DESCRIPTION
**Pull request by NehalDutta**
- This MR upgrades the Twilio Android SDK from 5.1.1 to 6.4.0. 
- Tested the changes for PCMT and Android PCM and they work fine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hrs-cordova-plugin-twilio",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (jnoel-hrs) [inline package.json]: Let's keep plugin versioning separate from PR changes to avoid conflicts

File: plugin.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
     xmlns:android="http://schemas.android.com/apk/res/android"
     id="hrs-cordova-plugin-twilio"
-    version="3.0.1">
+    version="3.0.2">
```
Comment (jnoel-hrs) [inline plugin.xml]: Let's keep plugin versioning separate from PR changes to avoid conflicts

**Review by jnoel-hrs — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NehalDutta) [reply package.json]: Sure. Changed it back to 3.0.1

File: plugin.xml (hunk 2 above)
Comment (NehalDutta) [reply plugin.xml]: Sure. Changed it back to 3.0.1

**Pull request merged**